libutils/json-utils.c

```diff
@@ -179,17 +179,15 @@ static char *filtered_copy(const char *src, char *dst)
  */
 bool JsonParseEnvFile(const char *input_path, size_t size_max, JsonElement **json_out)
```

Comment:
`const size_t size_max` might be better, but that's not really related to your PR.